server/app/views/admin/questions/QuestionConfig.java

```diff
@@ -79,8 +81,14 @@ public static Optional<DivTag> buildQuestionConfig(QuestionForm questionForm, Me
             config
                 .addMultiOptionQuestionFields((MultiOptionQuestionForm) questionForm, messages)
                 .getContainer());
+      case FILEUPLOAD:
+        return multipleFileUpload
```

Comment:
I would have expected a new screenshot for this change. Is it because screenshot coverage is missing, or browser tests run with feature flags off?

Reply:
Awesome, thank you!

And oh gosh, it's hidden in another mega-test just like the other ones. I'll look into this when I'm in on Monday to see if there is a good way to add something without copying the whole test, otherwise it might be best to just wait for the flag flip.

Reply:
Yeah looking at it since it tests all questions we have a similar choice to the one I made in the other test (either copy the whole set of tests, or pull out a file-upload specific version of it). In this case I'm a bit more inclined not to do anything since most of that test has nothing to do with file upload questions in particular, but keep me honest if you disagree. :)

Reply:
👍